src/components/Dashboard/TodoList.vue

```diff
@@ -1740,8 +1740,8 @@ export default class TodoList extends Vue {
 .todo-subtitle {
   color: $gray7;
   display: flex;
-  align-items: center;
   justify-content: flex-start;
+  flex-direction: column;
```

Comment:
I think this breaks some specific cases, like this:

![image](https://user-images.githubusercontent.com/10002889/197896680-0cb4ebde-1bed-4a33-a0e7-c9ce94891c19.png)

Check the code for status pending, status error, and status paid.

Reply:
In order not to have something like this, I used a different class `.todo-subtitle-warning` for the warning not in good standing instead. The class `.todo-subtitle` remains unchanged.

Reply:
Please put the 2 CSS selectors on separate lines and add a space between the key and val on line 1748.

Did you test with most scenarios?

Reply:
Yes, I tested this with most scenarios.